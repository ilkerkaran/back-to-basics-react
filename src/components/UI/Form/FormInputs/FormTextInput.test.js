import React from 'react';
import renderer from 'react-test-renderer';
import Text from './FormTextInput';


describe('<Text /> snapshot ', () => {
  const inputConfig = {
    inputType: 'text',
    inputName: 'text',
    label: 'Text',
    isRequired: true,
    register: () => {},
    errors: {}
  };


  it('renders correctly', () => {
    const tree = renderer
      .create(<Text inputConfig={inputConfig}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
