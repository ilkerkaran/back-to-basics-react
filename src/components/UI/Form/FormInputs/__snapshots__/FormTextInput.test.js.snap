// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text /> snapshot  renders correctly 1`] = `
<div>
  <input
    className="Input"
    name="text"
    placeholder="Text"
    type="text"
  />
  <div
    style={
      Object {
        "color": "darkred",
      }
    }
  />
</div>
`;
